components/category_menu_items/category_menu_items.tsx

```diff
@@ -83,11 +83,18 @@ const CategoryMenuItems = (props: Props): JSX.Element | null => {
     }
 
     const categoryMenuItems = filteredCategories.map((category: ChannelCategory) => {
+        let text = category.display_name;
+        if (category.type === CategoryTypes.FAVORITES) {
+            text = intl.formatMessage({id: 'sidebar.types.favorites', defaultMessage: 'Favorites'}).toLowerCase();
+        }
+        if (category.type === CategoryTypes.CHANNELS) {
+            text = intl.formatMessage({id: 'sidebar.types.channels', defaultMessage: 'Channels'}).toLowerCase();
```

Comment:
Also should we use [toLocaleLowerCase](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase) ?? @hmhealey

Reply:
@koox00 Should we use [toLocaleLowerCase](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase).

Reply:
cc @hmhealey too for escalation

Reply:
I don't think we need to change that right now. I don't think we've actually used that anywhere else in the code base.

That said, do we even need to make this lower case? Since it's all capitalized by the CSS, I don't think that would end up doing anything.

Reply:
@hmhealey Yeah toLowerCase is required as Translation is AllCaps and styling isn’t working on that.
<img width="403" alt="Screenshot 2022-03-22 at 9 40 31 AM" src="https://user-images.githubusercontent.com/96474572/159406473-42050725-f5f4-4596-9c42-18979327a9aa.png">

Reply:
Oh, I see. I didn't realize the strings for that were uppercase in the translation files. I think it'd probably make sense to change those to be properly capitalized in the translation files (like how you have the `defaultMessage` capitalized here) so that we can use them here without transforming them. We already use CSS to transform the headings in the sidebar to all caps, so no changes would be needed there

Reply:
@VishakhaPoonia I checked those are ALLCAPS in all the translations files, should I change the what you have suggested or just use toLowerCase?

Reply:
@VishakhaPoonia yes please change in translation itself